modules/imgproc/src/contours.cpp

```diff
@@ -1287,6 +1349,109 @@ typedef  struct CvLinkedRunPoint
 }
 CvLinkedRunPoint;
 
+#if CV_SSE2
+inline int findStartContourPoint(uchar *src_data, CvSize img_size, int j, bool haveSSE2) {
+#else
+inline int findStartContourPoint(uchar *src_data, CvSize img_size, int j) {
```

Comment:
here you can add parameter bool /* haveSSE2 */ to avoid conditional compilation when you call this function